### PR TITLE
fix: hide cancel button when order state is cancelled

### LINKED ISSUE #3719: cancel order only once
**Describe the bug**
once the order is cancelled , Cancel button is still enabled and on click shows the message "your order is cancelled" While it should be disable once order is cancelled ?

**To Reproduce**
Steps to reproduce the behaviour:
1. place a order 
2. cancel it and see it in http://localhost:4200/events/{{event_id}}/tickets/orders/cancelled
3. in this section one can again click on cancel order button
4. And the cancellation mail will send again and again to user and notification show up on notification board

**Expected behaviour**
one should able to cancel order only once .
Cancel order button should be disabled in that section .
**Screenshots**
Screenshot from 2019-12-23 00-36-18

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

I am working on this .

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #3719 

#### Short description of what this resolves:
don't show cancel button when order state is cancelled .
Screenshot from 2019-12-24 23-29-44
Screenshot from 2019-12-24 23-29-50

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by maze-runnar** [comment]
@uds5501 please review .

File: app/templates/components/ui-table/cell/events/view/tickets/orders/cell-order.hbs [hunk 1]
```diff
@@ -29,7 +29,7 @@
       <i class="check icon"></i>
     {{/ui-popup}}
   {{/if}}
-  {{#if (and (not-eq record.status 'cancelled') (not-eq record.status 'expired') (can-modify-order record))}}
+  {{#if (and (not-eq record.status 'cancelled') (not-eq record.status 'expired') (can-modify-order record) (not-eq extraRecords.status 'cancelled'))}}
```
Comment (kushthedude) [inline app/templates/components/ui-table/cell/events/view/tickets/orders/cell-order.hbs]: There is already `record.status`, Why to add `extraRecords.status` ?

Comment (maze-runnar) [reply app/templates/components/ui-table/cell/events/view/tickets/orders/cell-order.hbs]: i  am not able to understand what is use of "record.status" though it's not working . if it's working then cancel order button should not be functional in that section but it is .In the code every where else  `extraRecord` is being used for getting status.

Comment (kushthedude) [reply app/templates/components/ui-table/cell/events/view/tickets/orders/cell-order.hbs]:
```suggestion
  {{#if (and (not-eq extraRecords.status 'cancelled') (not-eq extraRecords.status 'expired') (can-modify-order record))}}
```
Change to this & check if it is working for expired orders too.

Comment (kushthedude) [reply app/templates/components/ui-table/cell/events/view/tickets/orders/cell-order.hbs]: Do this and check, If it is showing for cancelled then it must be showing for expired too
```suggestion
  {{#if (and (not-eq extraRecords.status 'cancelled') (not-eq extraRecords.status 'expired') (can-modify-order record))}}
```

**Comment by maze-runnar** [comment]
it's working now for expired and cancelled both
Screenshot from 2019-12-27 10-14-06
Screenshot from 2019-12-27 10-16-15

**Pull request merged**